app/src/main/touch-bar.js

```diff
@@ -3,29 +3,21 @@ import plugins from './plugins';
 
 const {TouchBarButton, TouchBarPopover} = TouchBar;
 
-const aspectRatioToSize = {
-  '16:9': '1600x900',
-  '5:4': '1280x1024',
-  '5:3': '1280x768',
-  '4:3': '1024x768',
-  '3:2': '480x320',
-  '1:1': '512x512',
-  Custom: 'Custom'
-};
+const aspectRatioToSize = new Map([
+  ['16:9', '1600x900'],
+  ['5:4', '1280x1024'],
+  ['5:3', '1280x768'],
+  ['4:3', '1024x768'],
+  ['3:2', '480x320'],
+  ['1:1', '512x512'],
+  ['Custom', 'Custom']
+]);
 
 const createAspectRatioPopover = ({onChange}) => {
-  const buttons = [
-    '16:9',
-    '5:4',
-    '5:3',
-    '4:3',
-    '3:2',
-    '1:1',
-    'Custom'
-  ].map(aspectRatio => {
+  const buttons = Array.from(aspectRatioToSize.keys()).map(aspectRatio => {
```

Comment:
Why not `[aspectRatioToSize.keys()...]`?

Reply:
No reason. Forgot that it's supported now :).